pandas/tseries/tests/test_resample.py

```diff
@@ -419,25 +419,32 @@ def test_agg_misc(self):
             assert_frame_equal(result, expected, check_like=True)
 
         # series like aggs
-        expected = pd.concat([t['A'].sum(),
-                              t['A'].std()],
-                             axis=1)
-        expected.columns = ['sum', 'std']
-
         for t in [r, g]:
-            result = r['A'].agg({'A': ['sum', 'std']})
+            result = t['A'].agg({'A': ['sum', 'std']})
+            expected = pd.concat([t['A'].sum(),
+                                  t['A'].std()],
+                                 axis=1)
+            expected.columns = ['sum', 'std']
+
+            assert_frame_equal(result, expected, check_like=True)
+
+            expected = pd.concat([t['A'].agg(['sum', 'std']),
+                                  t['A'].agg(['mean', 'std'])],
+                                 axis=1)
+            expected.columns = pd.MultiIndex.from_tuples([('A', 'sum'),
+                                                          ('A', 'std'),
+                                                          ('B', 'mean'),
+                                                          ('B', 'std')])
+            result = t['A'].agg({'A': ['sum', 'std'], 'B': ['mean', 'std']})
```

Comment:
It's just a bit strange and inconsistent that the key of the dict ('C') in [5] is completely ignored, while in [4] the keys of the dict are used as column names, while only the length of the dict differs

>  maybe we should raise here as the user could think they are actually operating on something they are not.

I would be +1 to raise here, but in the way it was in 0.17.1: raising for `g['D'].agg({'C': ['sum', 'std']})` but not for `g['D'].agg({'C': 'sum', 'D': 'std']})`